apps/sms/views/conversation/js/compose.js

```diff
@@ -314,7 +319,7 @@ var Compose = (function() {
   }
 
   var compose = {
-    init: function composeInit(formId) {
+    init(formId, mmsSizeLimitation, maxConcatenatedMessages) {
```

Comment:
I really like that we inject such properties from outside now \0/ Hope we don't lose a lot of time on two settings calls when we init this